File: coding/tasks/csv-summariser-python/test_solution.py
import sys

sys.path.insert(0, "/workspace")

from solution import sum_column


def test_column_sum() -> None:
    assert sum_column([["a", "1"], ["b", "2"], ["c", "3"]], 1) == 6.0
    assert sum_column([], 0) == 0.0
    assert sum_column([["x", "y"], ["1", "2"]], 0) == 1.0

File: coding/tasks/anagram-rust/test_solution.rs
#[path = "/workspace/solution.rs"]
mod solution;

use solution::is_anagram;

#[test]
fn detects_anagrams() {
    assert!(is_anagram("Listen", "Silent"));
    assert!(!is_anagram("hello", "world"));
    assert!(is_anagram("a", "a"));
}
